docs/examples/deployment_prediction_generation.md

```diff
@@ -27,12 +27,12 @@ This DAG will create a new deployment from a registered model and demonstrate ho
 4. Upload the dataset to be used for predictions
 5. Apply a wrangling recipe to the dataset
 6. Make predictions using the deployed model
-7. Wait for the predictions to be finish scoring
+7. Wait for the predictions to finish scoring
 
 ## Result
 
 You now have a new deployment created from a registered model and test making predictions from the new deployment.
 
 ## Troubleshooting
 In order to make predictions from a deployment via DataRobot's Prediction API, you need a prediction server ID.
-See https://docs.datarobot.com/en/docs/api/reference/predapi/pred-server-id.html for more information on how to get the prediction server ID for your organization or deployments.
+See (Get a prediction server ID)[https://docs.datarobot.com/en/docs/api/reference/predapi/pred-server-id.html]{ target=_blank } for more information on how to get the prediction server ID for your organization or deployments.
```

Comment:
Does target blank work for the generated docs? Also, in our repo usually the text is bracketed and the link is in parens. Not sure if these docs behave differently.